ProcessMaker/process-templates
# FOUR-9991: Some templates do not have the model view

## Issue and Steps to Reproduce: ##

Some templates do not have the model view.

- Log in 
- Click on Designer
- Click on +PROCESS

When we click on the following templates, the Model view is missing.

```
Change Request
Contract review
Net promoter score
```
![Screenshot 2023-08-24 at 10 03 08 AM](https://github.com/ProcessMaker/process-templates/assets/5769433/c5273a0f-0758-4fcd-a839-4b9482728c56)

## Expected Behavior: ##

All templates should have the Model design view.

## Solution ##
Added missing SVG to the templates in question.

## How to Test ##
- Please update the .env variable in Core to this branch. (FOUR-9991)
```
DEFAULT_TEMPLATE_REPO=process-templates
DEFAULT_TEMPLATE_BRANCH=bugfix/FOUR-9991
DEFAULT_TEMPLATE_CATEGORIES=all
```
- Please run the template sync command:  `php artisan processmaker:sync-default-templates`.
- Make sure all templates have the svg within the SVG column in process_templates table.
- Follow the steps to reproduce steps and make sure the following templates have the Model view.
```
Change Request
Contract review
Net promoter score
```

![Screenshot 2023-08-24 at 10 30 34 AM](https://github.com/ProcessMaker/process-templates/assets/5769433/f075131e-36ff-422d-8e8a-f2113f635bad)


## Related Tickets & Packages ##
Ticket [FOUR-9991](https://processmaker.atlassian.net/browse/FOUR-9991)

## Code Review Checklist



[FOUR-9991]: https://processmaker.atlassian.net/browse/FOUR-9991?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
